specification/schemas/shops/Shop.json

```diff
@@ -106,16 +106,28 @@
                 },
                 "tracking": {
                   "allOf": [
+                    {
+                      "$ref": "#/components/schemas/Style"
+                    },
                     {
                       "properties": {
                         "logo_url": {
                           "$ref": "#/components/schemas/ImageUrl"
+                        },
+                        "hide_customer_reference": {
+                          "type": "boolean",
+                          "example": false,
+                          "default": false,
+                          "description": "Whether to hide the customer reference from the shipment's tracking page."
+                        },
+                        "hide_sender_address": {
+                          "type": "boolean",
+                          "example": false,
+                          "default": false,
+                          "description": "Whether to hide the sender address from the shipment's tracking page."
```

Comment:
It seems that these properties will always be in the response. Can you make the spec reflect that by putting these in the `required` array?

Reply:
made boolean style properties (for both `returns` and `tracking`) `required` in the ShopResponse